src/sensors/RotatingRaySensor.cpp

```diff
@@ -0,0 +1,458 @@
+/*
+ *  RotatingRaySensor.cpp
+ *
+ *  Created by Stefan Haase, Kai von Szadkowski, Malte Langosz
+ *
+ */
+
+#include "RotatingRaySensor.hpp"
+
+#include <mars_interfaces/sim/NodeManagerInterface.h>
+#include <mars_interfaces/sim/SimulatorInterface.h>
+#include <mars_interfaces/sim/LoadCenter.h>
+
+#include <data_broker/DataBrokerInterface.h>
+#include <mars_interfaces/graphics/GraphicsManagerInterface.h>
+#include <cfg_manager/CFGManagerInterface.h>
+#include <mars_utils/MutexLocker.h>
+
+#include <cmath>
+#include <cstdio>
+#include <cstdlib>
+
+namespace mars {
```

Comment:
Please ensure opening braces to start in a new line (cf. style in files for other sensors)